libyul/backends/evm/NoOutputAssembly.cpp

```diff
@@ -30,25 +30,25 @@ using namespace langutil;
 using namespace yul;
 
 
-void NoOutputAssembly::appendInstruction(solidity::Instruction _instr)
+void NoOutputAssembly::appendInstruction(dev::eth::Instruction _instr)
 {
-	m_stackHeight += solidity::instructionInfo(_instr).ret - solidity::instructionInfo(_instr).args;
+	m_stackHeight += instructionInfo(_instr).ret - instructionInfo(_instr).args;
```

Comment:
Why doesn't `instructionInfo` need the namespace?

Reply:
Ah. Weird.

Reply:
Ah, it's an instance of Koenig lookup.

Reply:
Do you want me to qualify the access?

Reply:
Not needed